Sources/PointFree/Account/Change.swift

```diff
@@ -108,8 +98,24 @@ func requireActiveSubscription<A>(
       <| middleware
 }
 
+func subscriptionModificationErrorMiddleware<A>(
+  _ conn: Conn<StatusLineOpen, A>
+  ) -> IO<Conn<ResponseEnded, Data>> {
+
+  return conn |> redirect(
+    to: .account(.index),
+    headersMiddleware: flash(
+      .error,
+      """
+      We couldn’t modify your subscription at this time. Please try again or contact
```

Comment:
this flash message is getting used in a few spots now so I extracted the middleware out